### PR TITLE
Add patients, treatments and medical_histories table

### PR DESCRIPTION
**Pull request by Gdiazdiaz**
## Tasks completed

- Create Clinic DataBase.
- Add patients, medical_histories, treatments, invoices, and invoice items tables.
- Add many to many table for treatments and medical histories.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   type text,
+   name text
+);
+
+CREATE TABLE medical_histories(
+   id serial Primary key,
+   admitted_at timestamp,
+   patient_id integer References patients (id),
+	status text
+);
```
Comment (franclobo) [inline schema_based_on_diagram.sql:18]: - [ ] Please 🙏🏽 add the FK indexes. For example: `CREATE INDEX ON medical_histories (patient_id);`

This is a requirement of the project 👇🏽 

> Remember to add the FK indexes.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+CREATE TABLE invoices(
+   id serial Primary key,
+   total_amount decimal,
+   generated_at timestamp,
+   payed_at timestamp,
+   medical_history_id integer References medical_histories (id)
+);
```
Comment (franclobo) [inline schema_based_on_diagram.sql:28]: - [ ] Please 🙏🏽 add the FK indexes as requested above 👆🏽

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+CREATE TABLE invoice_items(
+   id serial Primary key,
+   unit_price decimal,
+   quantity integer,
+   total_price decimal,
+   invoice_id integer References invoices (id),
+   treatment_id integer References treatments (id)
+);
```
Comment (franclobo) [inline schema_based_on_diagram.sql:39]: - [ ] Please 🙏🏽 add the FK indexes as requested above 👆🏽

File: schema_based_on_diagram.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Create table med_hist_treat(
+   medical_history_id integer References medical_histories (id),
+   treatment_id integer References treatments (id)
+);
```
Comment (franclobo) [inline schema_based_on_diagram.sql:46]: - [ ] Please 🙏🏽 add the FK indexes as requested above 👆🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Gdiazdiaz, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of CREATE TABLE ✔️ 
- Correct use of JOIN TABLE ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🍾🎉**
Hello @Gdiazdiaz and @sambeck87 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**